test/rtos_drivers/hil_add/src/individual_tests/uart/uart_test.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 XMOS LIMITED.
```

Comment:
I will be slightly surprised if this copyright date passes the source file checker.

Reply:
That's because there are no source checkers in the SDK repos. Anyhow, fixed.